app/Commands/Format.hs

```diff
@@ -17,17 +18,32 @@ data FormatRenderMode
 data FormatTarget
   = TargetFile
   | TargetDir
+  | TargetStdin
 
 runCommand :: forall r. Members '[Embed IO, App, Resource, Files] r => FormatOptions -> Sem r ()
 runCommand opts = do
-  f <- filePathToAbs (opts ^. formatInput)
+  globalOpts <- askGlobalOptions
+  let isStdin = globalOpts ^. globalStdin
+  f <- sequence (filePathToAbs <$> (opts ^. formatInput))
+
   let target = case f of
-        Left {} -> TargetFile
-        Right {} -> TargetDir
+        Just (Left {}) -> TargetFile
+        Just (Right {}) -> TargetDir
+        Nothing -> TargetStdin
   runOutputSem (renderFormattedOutput target opts) $ runScopeFileApp $ do
     res <- case f of
-      Left p -> format p
-      Right p -> formatProject p
+      Just (Left p) -> format p
+      Just (Right p) -> formatProject p
+      Nothing ->
+        if isStdin
```

Comment:
Style: We prefer to use multiway-if rather than `if ... then ... else`.

i.e change to

```
if 
   | isStdin -> ...
   | otherwise -> ...
```

Reply:
Good to know! Changed to that 🙂 
Do we have a style guide on all these things I can check?

Reply:
Unfortunately we do not have anything written down. I think we should, as long as we make it clear that we always expect exceptions tot he guidelines. Let's discuss with the whole team.

Reply:
Sounds great!